tests/snuba/tsdb/test_tsdb_backend.py

```diff
@@ -395,7 +397,7 @@ def test_distinct_counts_series_users(self):
             == {}
         )
 
-    def get_distinct_counts_totals_users(self):
+    def test_get_distinct_counts_totals_users(self):
```

Comment:
this 6 year old test wasn't actually running because of the function name 😂 fixed!!